lerobot/templates/visualize_dataset_template.html

```diff
@@ -2,217 +2,334 @@
 <html lang="en">
 
 <head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <script src="https://cdn.jsdelivr.net/npm/dygraphs@2.1.0/dist/dygraph.min.js" type="text/javascript"></script>
-  <script src="https://cdn.tailwindcss.com"></script>
-  <title>{{ dataset_info.repo_id }} episode {{ episode_id }}</title>
-  <style>
-    /* css for graph legends */
-    #labels>span {
-      width: 14rem;
-      display: block;
-      font-size: 1rem;
-    }
-  </style>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/alpinejs/3.13.5/cdn.min.js" defer></script>
+    <script src="https://cdn.jsdelivr.net/npm/dygraphs@2.2.1/dist/dygraph.min.js" type="text/javascript"></script>
+    <script src="https://cdn.tailwindcss.com"></script>
+    <title>{{ dataset_info.repo_id }} episode {{ episode_id }}</title>
 </head>
 
-<body class="flex h-screen max-h-screen bg-slate-950 text-gray-200">
-  <!-- Sidebar -->
-  <div id="sidebar" class="w-60 bg-slate-900 p-5 break-words max-h-screen overflow-y-auto">
-    <h1 class="mb-4 text-xl font-semibold">{{ dataset_info.repo_id }}</h1>
-
-    <ul>
-      <li>
-        Number of samples/frames: {{ dataset_info.num_samples }}
-      </li>
-      <li>
-        Number of episodes: {{ dataset_info.num_episodes }}
-      </li>
-      <li>
-        Frames per second: {{ dataset_info.fps }}
-      </li>
-    </ul>
-
-    <p>Episodes:</p>
-    <ul class="ml-2">
-      {% for episode in episodes %}
-      <li class="font-mono text-sm mt-0.5">
-        <a href="episode_{{ episode }}" class="underline">
-          Episode {{ episode }}
-        </a>
-      </li>
-      {% endfor %}
-    </ul>
-
-  </div>
-
-  <!-- Toggle sidebar button -->
-  <button class="flex items-center opacity-50 hover:opacity-100 mx-1" onclick="toggleSidebar()" title="Toggle sidebar">
-    <div class="bg-slate-500 w-2 h-10 rounded-full"></div>
-  </button>
-
-  <!-- Content -->
-  <div class="flex-1 max-h-screen flex flex-col gap-4 overflow-y-auto">
-    <h1 class="text-xl font-bold mt-4 font-mono">
-      Episode {{ episode_id }}
-    </h1>
-
-    <!-- Videos -->
-    <div class="flex flex-wrap gap-1">
-      {% for video_info in videos_info %}
-      <div class="max-w-96">
-        <p class="text-sm text-gray-300 bg-gray-800 px-2 rounded-t-xl truncate">{{ video_info.filename }}</p>
-        <video type="video/mp4" class="min-w-64">
-          <source src="{{ video_info.url }}">
-          Your browser does not support the video tag.
-        </video>
-      </div>
-      {% endfor %}
-    </div>
+<body class="flex h-screen max-h-screen bg-slate-950 text-gray-200" x-data="createAlpineData()" @keydown.window="(e) => {
+    if (e.keyCode === 32 || e.key === ' ') {
+        e.preventDefault();
+        $refs.btnPause.classList.contains('hidden') ? $refs.btnPlay.click() : $refs.btnPause.click();
+      }
```

Comment:
I am wondering if we could use the bottom and top arrows to navigate between episodes?
I have a tool on google chrome to open a lot of tabs really easily, but most of the people will struggle to click on the episodes on the left.

The illustration of me opening a lot of tabs with my tool ^^
<img width="1013" alt="Screenshot 2024-07-30 at 11 08 55" src="https://github.com/user-attachments/assets/76fde6b6-ab84-4939-bb98-c9a4afc99492">

Also I am wondering how to convey that these shortcuts exist?

Importantly, can we autoplay when the episode page is ready so that we dont have to press space but just the arrows?

Thanks!

Reply:
with the recent changes:
1. the videos are autoplaying
2. you can use up/down arrow to switch between episodes


let me think about
> Also I am wondering how to convey that these shortcuts exist?

Reply:
<img width="500" alt="image" src="https://github.com/user-attachments/assets/c86e2e38-42a8-4108-83c0-1860da2b0665">

> Also I am wondering how to convey that these shortcuts exist?

I think simple text https://github.com/huggingface/lerobot/pull/346/commits/c6250b5c876232581890fb1108a783e66f923aab would be good for now